### PR TITLE
Fix #1191

### LINKED ISSUE #1191: Remove R5 dependencies org.osgi.framework.namespace and org.osgi.framework.wiring

### PR DESCRIPTION
**Pull request by adimova**
Signed-off-by: Ana Dimova <a.dimova@prosyst.bg>

### CODE REVIEW

File: bundles/automation/org.eclipse.smarthome.automation.providers/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -14,6 +14,7 @@ Import-Package:
  org.osgi.framework,
  org.osgi.framework.namespace,
```
Comment (BenediktNiehues) [inline bundles/automation/org.eclipse.smarthome.automation.providers/META-INF/MANIFEST.MF]: The package imports should be removed as well.

**Comment by maggu2810** [comment]
This comment is a little bit to late, because it has been merged now... sorry about that.

But IIRC you normally does not want commented code in the code base. Has this been accepted by this change by intention? Are there some conditions where it is good to leave old code in the source base. I do not complain about it, just want to know if something has changed.

**Comment by kaikreuzer** [comment]
You are right, this is normally not nice and no, we do not have any guidelines or official cases where such code should stay in the codebase.
@adimova maybe you can comment on why you left it in and whether it is ok to remove it?

**Comment by adimova** [comment]
I left it for the moment when we want to move to R5 verson

**Comment by kaikreuzer** [comment]
> when we want to move to R5 version

The code is still available in the git history, so it is not really required to keep it as a comment as well.

**Pull request merged**